viewing_party/party.py

```diff
@@ -1,23 +1,148 @@
 # ------------- WAVE 1 --------------------
 
+from symbol import subscriptlist
+
+
 def create_movie(title, genre, rating):
-    pass
+  new_move = {"title": title,
+              "genre" : genre,
+              "rating": rating}
+  for key, value in new_move.items():
+    if value == None:
+      return None
+  else:
+      return new_move
+
+def add_to_watched(user_data, movie):
+  user_data = {"watched":[movie]}
+  return user_data

```

Comment:
When you inspect user_data (as it is passed in from the test, we can see that user_data is already a dict with a key called "watched" (see lines 69-71 of test_wave_01.py:
```python 
user_data = {
    "watched": []
}
```

Since that's the case, when you reassign user_data to a dictionary literal on line 17 then you overwrite anything that was previously in the dictionary ("friends", "watchlist"). In this example, we have a pretty bare bones user_data so the test does pass, but if user_data had other key/value pairs, we don't want to overwrite them and lose the data.

Since "watched" is already list, you don't need to put `movie` into a list with `[movie]`.

We can just write: 
```python
def add_to_watched(user_data, movie):
    user_data{"watched"].append(movie)
    return user_data
```